plugin.audio.tripler/resources/lib/scraper.py

```diff
@@ -0,0 +1,2244 @@
+#!/usr/bin/env python
+import bs4, html, time, json, re, sys
+from concurrent.futures import ThreadPoolExecutor
+from datetime import datetime, timedelta
+# from marshmallow_jsonapi import Schema, fields
+
+
+IS_PY3 = sys.version_info[0] > 2
+if IS_PY3:
+    from urllib.request import Request, urlopen
+    from urllib.parse import parse_qs, urlencode
+    from urllib.error import URLError
+else:
+    from urllib2 import Request, urlopen
+    from urllib2 import urlencode
+    from urlparse import parse_qs
+
+
+DATE_FORMAT = '%Y-%m-%d'
+
+URL_BASE = 'https://www.rrr.org.au'
+
+USER_AGENT = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36'
+
+stderr = False
+ignore_on_air = False
+
+
+def get(resource_path):
+    return urlopen_ua(Scraper.url_for(resource_path))
+
+def urlopen_ua(url):
+    if stderr:
```

Comment:
There is no use for `stderr` in Kodi. Not even for debugging. Use `xbmc.log()` if needed.